src/diffchecker.js

```diff
@@ -1,26 +1,45 @@
+const Diff = require('diff');
+
+function normalizeLines(str) {
+  const strWithNewLine = str.endsWith('\n') ? str : str + '\n';
```

Comment:
When adding lines to the end of a file, the 'diff' package reads it as removing and adding  the last line even though there isn't any changes. This line is to ensure a `\n` at the end so the package can read it as unchanged line.